### PR TITLE
Update che4z-release-information.adoc

### PR DESCRIPTION
**Pull request by DStatWriter**
Update release notes for 0.8.2 and 0.8.3

Signed-off-by: Daniel Statham <Daniel.statham@broadcom.com>

### What does this PR do?
Updates release notes for LSP for Cobol

### What issues does this PR fix or reference?
#14903

### CODE REVIEW

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 1]
```diff
@@ -22,7 +22,15 @@ Eclipse Che4z currently comprises two products, z/OS Resource Explorer and COBOL
 
 == Release Notes - COBOL Language Support
 
-=== Version 0.81
+--- Version 0.8.3
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
=== Version 0.8.3
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 2]
```diff
@@ -22,7 +22,15 @@ Eclipse Che4z currently comprises two products, z/OS Resource Explorer and COBOL
...
+
+* Java automatic check fixed to recognize all supported JRE/JDK java implementations.
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc:27]:
```suggestion
* Java automatic check fixed to recognize all supported JRE and JDK implementations.
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 3]
```diff
@@ -22,7 +22,15 @@ Eclipse Che4z currently comprises two products, z/OS Resource Explorer and COBOL
...
+
+--- Version 0.8.2
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
=== Version 0.8.2
```

File: src/main/pages/che-7/extensions/che4z-release-information.adoc [hunk 4]
```diff
@@ -22,7 +22,15 @@ Eclipse Che4z currently comprises two products, z/OS Resource Explorer and COBOL
...
+
+* README Updated
```
Comment (rkratky) [inline src/main/pages/che-7/extensions/che4z-release-information.adoc]:
```suggestion
* README updated
```

**Review by rkratky — CHANGES_REQUESTED** [review]

**Comment by themr0c** [comment]
This still needs to have the commits signed off. Please, rebase, squash the commits, add your sign-off line to the commit message, and push --force.

**Comment by DStatWriter** [comment]
Closed for resubmission

**Pull request closed without merging**